<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>true_deposit</class>
 <widget class="QWidget" name="true_deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>True Deposit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Deposit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time (month)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Perecent (n%)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tax Rate (n%)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Periodicity of 
payments</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_per_pay_once">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Once a time</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Per_payments</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_per_pay_day">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Once a day</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Per_payments</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_per_pay_month">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Once a month</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Per_payments</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_per_pay_year">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Once a year</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Per_payments</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_cap_yes">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Kapitalization</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_cap_no">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Kapitalization</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Capitalization</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Refill</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_refill_no">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Refill</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_refill_month">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Once a month</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Refill</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_refill_year">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Once a year</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Refill</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Withdrawal</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_with_no">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>460</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_with_month">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>480</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Once a month</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_with_year">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>500</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Once a year</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_get_deposit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>10000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_get_time">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_get_percent">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_get_percent_tax">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_get_additional_refill">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>400</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_get_additional_withdrawal">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>210</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Get %</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>260</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SUM TAX</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>300</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Final deposit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_res_percents">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>230</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_res_taxes">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>270</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_res_deposit">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>310</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>450</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>490</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="Kapitalization"/>
  <buttongroup name="Per_payments"/>
  <buttongroup name="Refill"/>
 </buttongroups>
</ui>
